weaviate/collection/classes/filters.py

```diff
@@ -92,59 +92,90 @@ class _FilterValue(_Filters):
 
 
 class Filter:
+    """Define a filter based on a property to be used when querying a collection.
+
+    Use the `__init__` method to define the path to the property to be filtered on and then
+    use the methods of this class to define the condition to filter on.
+
+    See [the docs](https://weaviate.io/developers/weaviate/search/filters) for more details!
+    """
+
     def __init__(self, path: Union[str, List[str]], length: bool = False):
+        """Initialise the filter.
+
+        Arguments:
+            `path`
+                The path to the property to be filtered on. This can be a single string in the case of a
+                top-level property or a list of strings in the case of a nested cross-ref property.
+            `length`
+                If `True`, the length of the property will be used in the filter. This is only valid for
+                properties of type `string` or `text`. Defaults to `False`.
```

Comment:
Propose to add:

`And the inverted index is configured to index the property length.`